docs/resources/azure_storage_account.md

```diff
@@ -0,0 +1,132 @@
+---
+title: About the azure_storage_account Resource
+platform: azure
+---
+
+# azure_storage_account
+
+Use the `azure_storage_account` InSpec audit resource to test properties related to an Azure Storage Account.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_storage_account` resource block identifies an Azure storage account by `name` and `resource_group` or the `resource_id`.
+```ruby
+describe azure_storage_account(resource_group: 'rg', name: 'sa')  do
+  it { should exist }
+end
+```
+```ruby
+describe azure_aks_cluster(resource_id: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.ContainerService/managedClusters/{ClusterName}') do
```

Comment:
`azure_aks_cluster` --> `azure_storage_account `

Reply:
the resource_id should also be updated

Reply:
Thanks, @rmoles!
Fixed now.